src/components/Reports/ReportsFilters/ReportsFilters.test.js

```diff
@@ -61,4 +61,32 @@ describe('ReportsFilters component', () => {
     // Assert
     expect(container).not.toContainHTML('reports_filters.verified_domain');
   });
+
+  it('should show 2 and 3 weeks', async () => {
```

Comment:
What do you think about also adding the other scenario?

```js
   it('should show 1 week by default', async () => {
```

Reply:
yes i added now! thanks